build_runner/lib/src/asset/cache.dart

```diff
@@ -4,11 +4,10 @@
 import 'dart:async';
```

Comment:
Right now this one depends on details like `InputSet` from `build_runner`. Depending on whether we actually need that full interface in the places we're using this cache today we might need to have 2 copies.

I'll look at whether we should re-add a cache in `build` or add a different implementation only the package dealing with bazel in a followup